blockchain/indexers/slpindex.go

```diff
@@ -879,7 +879,7 @@ func DropSlpIndex(db database.DB, interrupt <-chan struct{}) error {
 		return err
 	}
 
-	return dropIndex(db, slpIndexKey, slpIndexName, interrupt)
+	return nil
```

Comment:
NIT: This function can just be:

```
func DropSlpIndex(db database.DB, interrupt <-chan struct{}) error {
  return dropIndex(db, slpIndexKey, slpIndexName, interrupt)
}
```

Reply:
good call!!! I overlooked that.  And done.